src/components/noteCard/noteCard.jsx

```diff
@@ -23,9 +27,11 @@ export const NoteCard = (props) => {
             <div className='notecard-section'>
                 <p className='notecard--date'>Created on 01/02/2020</p>
                 <div className="notecard--icons">
-                    <FiEdit className='note--icon' />
-                    <FiArchive className='note--icon' />
-                    <MdDelete className='note--icon' onClick={() => moveNoteToTrash(noteInfo)} />
+                    <MdDelete className='note--icon' title='trash' onClick={() => transferToTrash(noteInfo)} />
+                    {/* <BsTrash className='note--icon' title='trash' onClick={() => transferToTrash(noteInfo)} /> */}
```

Comment:
If commented code is not required do delete them

Reply:
ok thank yo, i will remove that